save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt

```diff
@@ -0,0 +1,235 @@
+                column(
+                    id = "organization",
+                    header = "Organization",
+                    accessor = { name }
+                ) { cellProps ->
+                    val organizationName = cellProps.value
+
+                    Fragment.create {
+                        td {
+                            Link {
+                                to = "/organization/$organizationName/tools"
+                                +organizationName
+                            }
+                        }
+                    }
+                }
+                column(
+                    id = "description",
+                    header = "Description",
+                    accessor = { description }
+                ) { cellProps ->
+                    Fragment.create {
+                        td {
+                            +(cellProps.value.orEmpty())
+                        }
+                    }
+                }
+                column(id = DELETE_BUTTON_COLUMN_ID, header = EMPTY_COLUMN_HEADER) { cellProps ->
+                    Fragment.create {
+                        td {
+                            deleteButton {
+                                val organization = cellProps.value
+                                val organizationName = organization.name
+
+                                id = "delete-organization-$organizationName"
+                                classes = deleteButtonClasses
+                                tooltipText = "Delete the organization"
+                                elementChildren = { childrenBuilder ->
+                                    with(childrenBuilder) {
+                                        fontAwesomeIcon(icon = faTrashAlt, classes = deleteIconClasses.joinToString(" "))
+                                    }
+                                }
+
+                                confirmDialog = ModalDialogStrings(
+                                    title = "Delete Organization",
+                                    message = """Are you sure you want to delete the organization "$organizationName"?""",
+                                )
+                                action = deleteOrganization(organization)
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        initialPageSize = INITIAL_PAGE_SIZE,
+        useServerPaging = false,
+        usePageSelection = false,
+        getAdditionalDependencies = { tableProps ->
+            /*-
+             * Necessary for the table to get re-rendered once an organization
+             * gets deleted.
+             *
+             * The order and size of the array must remain constant.
+             */
+            arrayOf(tableProps)
+        }
+    )
+
+    init {
+        state.organizations = mutableListOf()
+    }
+
+    override fun componentDidMount() {
+        super.componentDidMount()
+
+        scope.launch {
```

Comment:
you can make a request and set state directly in the FC

Reply:
check `useRequest {}`

Reply:
Will refactor and implement in a separate PR once there's a clear understanding how to use `tableComponent() `together with `FC<*>`.